packages/http-client-csharp/eng/scripts/CadlRanch-Helper.psm1

```diff
@@ -0,0 +1,46 @@
+$repoRoot = Resolve-Path (Join-Path $PSScriptRoot '..')
+
+function IsGenerated {
+    param (
+        [string]$dir
+    )
+
+    if (-not ($dir.EndsWith("Generated"))) {
+        return $false
+    }
+
+    $csFiles = Get-ChildItem -Path $dir -Filter *.cs -File
+    return $csFiles.Count -gt 0
+}
+
+function Capitalize-FirstLetter {
```

Comment:
Maybe we can use TextInfo.ToTitleCase

Reply:
But less of an issue if we don't need to export this function.

Reply:
yeah ill skip export